transport/grpc/config_test.go

```diff
@@ -482,7 +482,7 @@ func TestTransportSpec(t *testing.T) {
 				if tt.wantInbound.ClientMaxRecvMsgSize > 0 {
 					assert.Equal(t, tt.wantInbound.ClientMaxRecvMsgSize, inbound.t.options.clientMaxRecvMsgSize)
 				} else {
-					assert.Equal(t, defaultClientMaxRecvMsgSize, inbound.t.options.clientMaxRecvMsgSize)
+					assert.Equal(t, 1024*1024*64, inbound.t.options.clientMaxRecvMsgSize)
```

Comment:
nit: why did you change it from variable to magic number?

Reply:
Not a magic number :-)

Reply:
any number is magic number :) However,  shouldn't this work without making this change since the value of Maxsize is already updated?

Reply:
Does this [comment](https://github.com/yarpc/yarpc-go/pull/2151#discussion_r944136876) justify?

Reply:
Shouldn't we just fix the test instead? 
this is just a nit, I will leave it to you.